### PR TITLE
refactor: Fix configuration properties for dialects

### PR DESCRIPTION
**Pull request by Nurkambay**

### CODE REVIEW

**Comment by Nurkambay** [comment]
> Technically, it is not a user-visible bug fix, but a kind of refactoring. Please, change the commit message prefix.

Actually, after that fix user can use different configs for different dialects, for now it is not working :-(

**Comment by temanbrcom** [comment]
> > Technically, it is not a user-visible bug fix, but a kind of refactoring. Please, change the commit message prefix.
> 
> Actually, after that fix user can use different configs for different dialects, for now it is not working :-(

My point is that this feature has not been released yet, so it is not a fix of something in production.

**Review by temanbrcom — APPROVED** [review]
🦓

File: clients/cobol-lsp-vscode-extension/src/services/Settings.ts [hunk 1]
```diff
@@ -64,19 +64,19 @@ export class SettingsService {
...
      * @returns a list of local path
      */
-    public static getCopybookLocalPath(cobolProgramName: string, dialectType: string): string[] {
+    public static getCopybookLocalPath(cobolFileName: string, dialectType: string): string[] {
         if (dialectType !== "COBOL") {
-            const pathList: string[] = vscode.workspace.getConfiguration(SETTINGS_CPY_SECTION).get(`${PATHS_LOCAL_KEY}.${dialectType}`);
+            const pathList: string[] = vscode.workspace.getConfiguration(SETTINGS_CPY_SECTION).get(`${PATHS_LOCAL_KEY}.${dialectType.toLowerCase()}`);
             if (pathList && pathList.length > 0) {
-                return SettingsService.evaluateVariable(pathList, "program_name", cobolProgramName);
+                return SettingsService.evaluateVariable(pathList, "program_name", cobolFileName);
```
Comment (grianbrcom) [inline clients/cobol-lsp-vscode-extension/src/services/Settings.ts:75]: So, the local variable name looks correct now, but how is it about "program_name"? We can discuss it on the standup.

**Review by grianbrcom — APPROVED** [review]
:panda_face:

File: clients/cobol-lsp-vscode-extension/src/services/Settings.ts (hunk 1 above)
Comment (Nurkambay) [reply clients/cobol-lsp-vscode-extension/src/services/Settings.ts:75]: Yep, it is really confusing

File: clients/cobol-lsp-vscode-extension/package.json [hunk 2]
```diff
@@ -119,6 +119,22 @@
                     "description": "Default list of relative local paths to search for copybooks",
                     "uniqueItems": true
                 },
+                "cobol-lsp.cpy-manager.paths-local.maid": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "description": "Default list of relative local paths to search for copybooks for maid dialect",
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/package.json:127]: Seems MAID and IDMS should be in the upper case
@zeibura

**Review by grianbrcom — APPROVED** [review]
:sewing_needle:

**Review by zeibura — APPROVED** [review]

**Pull request merged**